admin/views/school/school-home.ejs

```diff
@@ -38,57 +46,72 @@
             </li>
 
             <li>
-                <% if (pinGenerationEligibilityData.isFamiliarisationPinGenerationAllowed) { %>
+                <% if (featureEligibilityData.isFamiliarisationPinGenerationAllowed) { %>
                 <h2 class="heading-small"><a href="/pupil-pin/generate-familiarisation-pins-overview">Try it out - password and PINs</a></h2>
                 <p>Generate school password and personal identification numbers (PINs) so pupils can familiarise themselves with the MTC</p>
                 <% } else { %>
                 <h2 class="heading-small"><a class="font-greyed-out disabled-link" href="#">Try it out - password and PINs</a></h2>
                 <span>Generate school password and personal identification numbers (PINs) so pupils can familiarise themselves with the MTC</span>
                 <p>
                     <strong class="disabled-banner">UNAVAILABLE</strong>
-                    <% if (pinGenerationEligibilityData.isWithinFamiliarisationUnavailableHours) { %>
+                    <% if (featureEligibilityData.isWithinFamiliarisationUnavailableHours) { %>
                     <span class="font-xsmall font-greyed-out">Open 8am - 4pm</span>
                     <% } %>
-                    <% if (pinGenerationEligibilityData.isFamiliarisationInTheFuture) { %>
-                    <span class="font-xsmall font-greyed-out">Open 8am - 4pm on <%= pinGenerationEligibilityData.familiarisationCheckStartDate.format('D MMMM YYYY')%> - <%= pinGenerationEligibilityData.familiarisationCheckEndDate.format('D MMMM YYYY') %></span>
+                    <% if (featureEligibilityData.isFamiliarisationInTheFuture) { %>
+                    <span class="font-xsmall font-greyed-out">Open 8am - 4pm on <%= featureEligibilityData.familiarisationCheckStartDate %> - <%= featureEligibilityData.familiarisationCheckEndDate %></span>
```

Comment:
much nicer than inline formatting 💯

Reply:
@GuyHarwood  I split the implementation into several functions. Have another look please.on `school-home-feature-eligibility-presenter.js`